testcases/PowerNVDump.py

```diff
@@ -135,7 +136,25 @@ def setup_test(self, dump_place="local"):
                 "ls -l /var/crash | grep '^d'| awk '{print $9}'")
         if dump_place == "net":
             self.crash_content = self.c.run_command(
-                "ssh root@%s \"ls -l %s | grep '^d'\" | awk '{print $9}'" % (self.dump_server_ip, self.dump_path))
+                "ssh -i %s root@%s \"ls -l %s | grep '^d'\" | awk '{print $9}'" % (self.rsa_path, self.dump_server_ip, self.dump_path))
+            print("crash content is %s"%(self.crash_content))
```

Comment:
please move this in in as debug log and even consider this to apply in test case 

can be like this 
```
log = OpTestLogger.optest_logger_glob.get_logger(__name__)
log.debug("crash content is %s"%(self.crash_content))
```